lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
         callback(null, json);
```

Comment:
I'm ok with what you decide, just wanted to point that we could be risking fixing the crash here, and crashing elsewhere due to whatever code not expecting a pure `null`. But that will probably be no problem :+1: